src/container/simulation.jl

```diff
@@ -56,7 +74,7 @@ function to_message_package(message_tuple::Tuple{Any, AbstractDict}, simulation_
     return MessagePackage(sender_aid, receiver_aid, simulation_time, message_tuple)
 end
 
-function to_cs_input(message_queue::ConcurrentQueue{Tuple{Any,AbstractDict,DateTime}})::Vector{MessagePackage}
+function to_cs_input!(message_queue::ConcurrentQueue{Tuple{Any,AbstractDict,DateTime}})::Vector{MessagePackage}
```

Comment:
`ConcurrentQueue{Tuple{Any,AbstractDict,DateTime}}`

Do we gain anything by moving `Tuple{Any,AbstractDict,DateTime}` into its own struct for readability here and in other places this is used?